sdk/tools/openrewrite-sample-compiler-ci.yml

```diff
@@ -0,0 +1,68 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - /sdk/tools/azure-openrewrite/
+      - /sdk/tools/azure-openrewrite-compiler-maven-plugin/
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages: 
+      - stage: CompileSamples
+        variables: 
+          - template: /eng/pipelines/templates/variables/globals.yml
+          - template: /eng/pipelines/templates/variables/image.yml
+
+        jobs:
+          - job: OpenRewriteCompilerPlugin
+            pool: 
+              name: $(LINUXPOOL)
+              image: $(LINUXVMIMAGE)
+              os: linux
+            displayName: Build and Run OpenRewrite Compiler Plugin
+            timeoutInMinutes: 60
+            steps:
+              - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                parameters:
+                  Paths:
+                    - /sdk/
+                    - /eng/
+                    - /common/perf-test-core/
+
+              - task: PowerShell@2
```

Comment:
Should this be changed to a `Maven@3` task?

Reply:
@srnagar I don't think so since it makes use of a PowerShell variable and has an inner command to convert the list into comma-separated input. I think it's more concise and readable as PowerShell.